foreman/api/host.go

```diff
@@ -72,9 +72,7 @@ type ForemanHost struct {
 	// ID of the hardware model
 	ModelId *int `json:"model_id,omitempty"`
 	// Whether or not to Enable BMC Functionality on this host
-	EnableBMC bool `json:"-"`
-	// Boolean to track success of BMC Calls
-	BMCSuccess bool `json:"-"`
+	EnableBMC bool `json:"enable_bmc,omitempty"`
```

Comment:
```suggestion
	EnableBMC bool `json:"-"`
```

It's not in the foreman API so you should keep the attribute that says so...